### PR TITLE
Enhancement/allegations_erath_pd

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: .wrgl/config.yaml [hunk 1]
```diff
@@ -1078,6 +1090,10 @@ branch:
     file: data/fuse/com_west_monroe_pd.csv
     primaryKey:
       - allegation_uid
+  com-erath-pd:
+    remote: com-erath-pd
+    merge: refs/heads/main
+    file: data/fuse/com_erath_pd.csv
```
Comment (pckhoi) [inline .wrgl/config.yaml:1096]: Do these new tables not have primary key?

Comment (ayyubibrahimi) [reply .wrgl/config.yaml:1096]: My bad. Improper merge. Fixing now.

**Comment by ayyubibrahimi** [comment]
@pckhoi - Apologies. Please review.

**Review by pckhoi — APPROVED** [review]
LGTM

**Pull request merged**